phpBB/docs/events.md

```diff
@@ -46,6 +46,18 @@ acp_email_find_username_append
 * Since: 3.1.7-RC1
 * Purpose: Add content at the end of the fimd username link
 
+acp_ext_list_disabled_name_after
```

Comment:
I don't think we need an extra script to take care of that. It just improves the readability when searching for events which is why it's always good to ensure alphabetic order when adding new events.